core/lexicon/en/plugin.inc.php

```diff
@@ -37,7 +38,7 @@
 $_lang['plugin_event_msg'] = 'Select the events that you would like this plugin to listen to.';
 $_lang['plugin_event_plugin_remove_confirm'] = 'Are you sure you want to remove this plugin from this event?';
 $_lang['plugin_event_update'] = 'Update Plugin Event';
-$_lang['plugin_lock'] = 'Lock plugin for editing';
+$_lang['plugin_lock'] = 'Locked for editing plugin';
```

Comment:
'Plugin is locked for editing' would read better. This is grammatically unclear.

Reply:
Thank you for notice. And yes, checkbox should describe state instead an action. It's why I changed it.